sample/src/main/java/com/tom_roush/pdfbox/sample/MainActivity.java

```diff
@@ -306,6 +309,25 @@ public void createEncryptedPdf(View v)
         }
     }
 
+    public void extractImgFromPdf(View v){
+        try{
+            ImageView imageView = (ImageView) findViewById(R.id.renderedImageView);
+            PDDocument document = PDDocument.load(assetManager.open("big_first.pdf"));
```

Comment:
This PDF file doesn't exist in the repository, can you change it to one that exists in the repository or remove the new sample function?

Reply:
If you need to test the sample function, you can download the "big_first.pdf" from the issue: [#379] .  
There is two commits in my pull request:   
the [19d94c3](https://github.com/TomRoush/PdfBox-Android/pull/380/commits/19d94c34e270b7178a82dfa89f4ab671a778747a) amis to fix the bug, and you can accept this commit.  
**the [d210b7a](https://github.com/TomRoush/PdfBox-Android/pull/380/commits/d210b7abf90e251088e2456c5c6bacb239604364) is used to reproduce the problem, and should not be accept.**
Thank you~